Adds sanity check for the term type inside the command manager (#1315)
isaac-sim/IsaacLab: source/extensions/omni.isaac.lab/docs/CHANGELOG.rst

Changelog
---------

0.27.8 (2024-11-01)

> ~~~~~~~~~~~~~~~~~~~
> Fixed

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~~

Fixed
```

Changelog
---------

0.27.6 (2024-10-28)
~~~~~~~~~~~~~~~~~~~

Added
^^^^^

> * Added sanity check if command term is valid type.
> 
> Fixed
> ^^^^^
> 
> * Fixed correct iteration over group_cfg_items
> 

Reviewer: ```suggestion
* Added sanity check if the term is a valid type inside the command manager.

Fixed
^^^^^

* Corrected the iteration over ``group_cfg_items`` inside the observation manager.

```

* Added frozen encoder feature extraction observation space with ResNet and Theia

> 0.27.6 (2024-10-28)

Reviewer: ```suggestion
0.27.6 (2024-10-25)
```
